frontend/src/FullPageRoutes/Matching.tsx

```diff
@@ -84,8 +84,15 @@ function MatchControls({
         title="Report"
         description="Optional report message"
         isModalOpen={showReport}
-        handleConfirm={() => onReport(matchId, description)}
-        handleCancel={() => setShowReport(false)}
+        handleConfirm={() => {
+          setDescription("")
+          setShowReport(false)
+          onReport(matchId, description)
```

Comment:
this should probably come before `setDescription("")`

Reply:
why? Pretty sure the order doesn't matter. afaik the state won't change until after the function is finished executing

Reply:
And the data in the request appears to confirm my assumption. description is correctly set when making the request.

Reply:
oh ok, I didn't realize that

Reply:
I think it would be more accurate to say the state variables update when the component re-renders.